timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -47,14 +51,26 @@ def __init__(
         self.feat_static_cat = feat_static_cat
         self.feat_static_real = feat_static_real
 
+        self.int_dtype = int_dtype
+        self.float_dtype = float_dtype
         if self.feat_static_cat is not None:
-            self.feat_static_cat = self.feat_static_cat.astype(np.int64)
+            self.feat_static_cat = self.feat_static_cat.astype(self.int_dtype)
```

Comment:
As far as I know, torch always uses int64 (and so did MXNet), so this might be redundant